### ISSUE TITLE
Product Query [3]: [Variation] `Products by Attribute` block

### ISSUE THREAD

**Issue by sunyatasattva** [woocommerce/woocommerce-gutenberg-products-block#7323]
Implement a “Products by Attribute” variation of the “Product Query” block. This variation should show at least the following settings:

* Attributes picker
* Stock status
* Order by

This is blocked by #7218 and #7322

**Comment by sunyatasattva** [comment]
Closing as won't do as of this comment: pdnLyh-2jZ-p2#comment-1685
